site/content/entry/citra-progress-report-2017-august.md

```diff
@@ -0,0 +1,159 @@
+tags = [ "progress-report" ]
+author = "anodium"
+coauthor = "saphiresurf"
+forum = 0000
++++
+
+2017 has been an amazing year, with more work having been put into the project
+than ever before, but it's not over yet! Last we met was
+[June](/entry/citra-progress-report-2017-june/), and just two months later the
+[Citra issue tracker](https://github.com/citra-emu/citra/pulls) is brimming with
+lots of changes once more. I am extremely excited for this month (and what might
+be of the next month or two) but we're getting ahead of ourselves! On this
+progress report, let's check out the big fish in the July and August pond of patches!
+
+## Updating The Software Renderer ([This](https://github.com/citra-emu/citra/pull/2766), [that](https://github.com/citra-emu/citra/pull/2822), [here](https://github.com/citra-emu/citra/pull/2871), [there](https://github.com/citra-emu/citra/pull/2872), and [those](https://github.com/citra-emu/citra/pull/2891)) by [wwylele](https://github.com/wwylele) and [Subv](https://github.com/Subv)
+
+Citra has two main rendering backends, software and OpenGL, but until very
+recently, no one gave any attention to the software backend, as OpenGL was faster,
+made better use of the GPU, and allowed things such as texture forwarding for
+higher resolution rendering. But there is one thing on which the OpenGL backend
+falls flat on its face; accuracy.
+
+But, [wwylele](https://github.com/wwylele) has just revived the software renderer
+(inspired by an attempt by [Subv](https://github.com/Subv) made prior), adding
+almost all of the features the hardware renderer had received over the years.
+In fact, every addition to the hardware renderer has been given an equivalent
+in software:
+
+{{% table %}}
+|      Feature      |                        Hardware                       |                                                   Software                                                   |
+|:-----------------:|:-----------------------------------------------------:|:------------------------------------------------------------------------------------------------------------:|
+| Fragment Lighting | [#1264](https://github.com/citra-emu/citra/pull/1264) |  [#2766](https://github.com/citra-emu/citra/pull/2766) [#2822](https://github.com/citra-emu/citra/pull/2822) |
+|   Spot Lighting   | [#2727](https://github.com/citra-emu/citra/pull/2727) |                             [#2871](https://github.com/citra-emu/citra/pull/2871)                            |
+|  Geometric Factor | [#2776](https://github.com/citra-emu/citra/pull/2776) |                             [#2872](https://github.com/citra-emu/citra/pull/2872)                            |
+|    Bump Mapping   | [#2762](https://github.com/citra-emu/citra/pull/2762) |                             [#2891](https://github.com/citra-emu/citra/pull/2891)                            |
+{{% /table %}}
+
+Although most users won't benefit much from this in the present, it is important
```

Comment:
Probably worth still pointing out that it's still incredibly slow and not for end users. You say above "OpenGL was faster...", which implies that this may no longer be the case. Software renderer is still farrrrr from usability par with the OpenGL renderer.